src/js/components/Page/Page.js

```diff
@@ -0,0 +1,36 @@
+import React, { useContext, useMemo } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Box } from '../Box';
+import { PageContext } from './PageContext';
+import { ResponsiveContext } from '../../contexts/ResponsiveContext';
+import { PagePropTypes } from './propTypes';
+
+const Page = ({ kind, ...rest }) => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  const value = useMemo(
+    () => ({
+      alignSelf: theme.page[kind].align,
+      fill: 'horizontal',
+      width: theme.page[kind].width,
+      pad: theme.page[kind].pad[size],
+    }),
+    [theme, size, kind],
+  );
+
+  return (
+    <PageContext.Provider value={value}>
+      <Box
+        overflow="horizontal"
```

Comment:
Seems like some of this stuff should be overrideable from props. This one seems a bit limiting.

Reply:
Could we do something like:

```
const Page = ({ kind, overflow = "horizontal", ...rest }) => {
    ...
    <PageContext.Provider value={value}>
      <Box
        overflow={overflow}
     ...
```

That way we'd still get a smart default (since we are putting a min-width constraint), but the default can still be overridden.

Reply:
I got rid of the overflow="horizontal" in the latest commit, but the user can set this by passing props. I also added an example of the user overriding the default `fill: 'horizontal'` prop in the Page/Narrow story.

Reply:
I see that now. 

I think there is still value in having a "smart default" to accommodate the min-width scenario. But perhaps that could come from the theme as well.